.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
```

Comment:
the conflict it resolved but I didn't squash your review changes, hoping that is okay and we can just proceed and carry along the extra commit history which shouldn't cause any issues.